test/Constraints/add_with_nil.swift

```diff
@@ -2,5 +2,5 @@
 
 func test(_ x: Int) -> Int {
   return x + nil
-  // expected-error@-1 {{cannot convert value of type 'Int' to expected argument type '_.Stride'}}
+  // expected-error@-1 {{type of expression is ambiguous without more context}}
```

Comment:
@xedin I noticed this diagnostic did change. Are you OK with this? The test case was originally added when a crash was fixed. I don't think the old diagnostic actually gave you any useful information (the real problem is there's no suitable overload of `+` -- as far as I can see the appearance of Stride in the old message is an emergent behavior of CSDiag and CalleeCandidateDiagnostics)